src/lib/hash/streebog/streebog.h

```diff
@@ -0,0 +1,71 @@
+/*
+* Streebog
+* (C) 2017 Ribose Inc.
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#ifndef BOTAN_STREEBOG_H__
+#define BOTAN_STREEBOG_H__
+
+#include <botan/hash.h>
+
+namespace Botan {
+
+/**
+* Streebog
```

Comment:
Can you add some brief comment here on the origin of the hash (eg refer to RFC, the GOST standard, etc)